### DISCUSSION TITLE
Any document to read about __syncthreads_and

### DISCUSSION THREAD

**Discussion by zhaolianshuizls** [NVIDIA/cutlass#1501] [Q&A] [answered]
https://github.com/NVIDIA/cutlass/blob/5c447dd84f8ae0e1d48ff9a2eae26ce8c4958101/include/cutlass/semaphore.h#L92, it looks like a very easy instruction, but it would be really nice if we can read some document about it.

**Comment by thakkarV** [comment, marked as answer]
This is a part of CUDA, and is documented already in https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#synchronization-functions
